Interop/ModelCard/src/main/java/org/tribuo/interop/modelcard/ModelCard.java

```diff
@@ -33,6 +33,7 @@
 
 /**
  * ModelCard feature to allow more transparent model reporting.
+ * Refer to <a href="https://arxiv.org/abs/1810.03993">Model Cards for Model Reporting</a> to read the original paper.
```

Comment:
Can you use this citation style here and in the package-info - https://github.com/oracle/tribuo/blob/main/Classification/XGBoost/src/main/java/org/tribuo/classification/xgboost/XGBoostClassificationTrainer.java#L51. You can put a link in as well, but I prefer to have a more academic citation style for references than a plain link.